packages/sample-apps/windows/SampleAppCPP/SampleAppCpp.vcxproj

```diff
@@ -177,7 +177,7 @@
     </Page>
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="boost" Version="1.76.0.0" />
+    <PackageReference Include="boost" Version="1.80.0.0" />
```

Comment:
>"1.80.0.0" [](http://example.com/codeflow?start=46&length=10)

Given the sizeable number of .vcxproj files this appears in, should this version number be contained in a shared .props file?

Reply:
That's a pending task for most dependencies.